.github/workflows/tests-e2e.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: '7.4'
-          coverage: none
+          snapshots: none
```

Comment:
Why change this?

Reply:
Whoops, search/replace mistake. This should of course not be changed here.